server/swagger.json

```diff
@@ -3999,6 +3999,13 @@
 									"$ref": "#/definitions/DashboardQuery"
 								}
 							},
+              "axes": {
+                "description": "The viewport for a cell's visualizations",
+                "type": "object",
+                "additionalProperties": {
```

Comment:
I think we'll only have `x`, `y` and `y2`.  Rather than the additionalProperties, I think this should be explicitly written in the documentation.

Also, the swagger JSON formatting is not consistent.  Perhaps prettier it?